### PR TITLE
Release v0.2.3 simpod-json-datasource

### PR DESCRIPTION
**Pull request by simPod**
- Read the [Review Guidelines](http://docs.grafana.org/plugins/developing/plugin-review-guidelines/) before submitting your plugin. These guidelines determine if the plugin is ready to be published or not.

- Commercial plugins require a plugin subscription to be published. Commercial plugin subscriptions help us fund continued development of our open source platform and software. See the [terms](https://grafana.com/terms) for more details.

- If possible, for datasource plugins please provide a description on how to set up a simple test environment. A docker container or simple install script helps speed up the review process a lot.

**REMOVE THE TEXT ABOVE BEFORE CREATING THE PULL REQUEST**

### CODE REVIEW

**Comment by simPod** [comment]
I think your CI is broken

```
Linting simpod-json-datasource version 0.2.3
https://github.com/simPod/grafana-json-datasource a380d96e1c228849562bef14de3b556557f86f86
Error
Failed to find README.md: NotFoundError: Could not find README.md
It seems, plugin isn't built properly: plugin.json found in src/ only
Version in repo.json (0.2.3) doesn't match plugin.json (%VERSION%)
```

https://github.com/simPod/GrafanaJsonDatasource/blob/a380d96e1c228849562bef14de3b556557f86f86/README.md

> It seems, plugin isn't built properly: plugin.json found in src/ only
?

> Version in repo.json (0.2.3) doesn't match plugin.json (%VERSION%)
this used to work, there's no reason to specify versions twice.

**Comment by marcusolsson** [comment]
Hi @simPod,

Since your last plugin update, we've updated the submission process.

Could you please update your version entry to match the [following format](https://github.com/grafana/grafana-plugin-repository#add-a-plugin-to-repojson):

{
  "id": "briangann-gauge-panel",
  "type": "panel",
  "url": "https://github.com/briangann/grafana-gauge-panel",
  "versions": [
    {
      "version": "0.0.8",
      "url": "https://github.com/briangann/grafana-gauge-panel",
      "download": {
        "any": {
          "url": "https://github.com/briangann/grafana-gauge-panel/releases/download/v0.0.8/briangann-gauge-panel-0.0.8.zip",
          "md5": "782c973460f330287b7efca5486aa015"
        }
      }
    }
  ]
}

In short, instead of references to commits, you'll need to submit a URL to the packaged plugin (.zip). You can read more on how to package (and sign) your plugin here:

- [Sign a plugin](https://grafana.com/docs/grafana/latest/developers/plugins/sign-a-plugin)
- [Package a plugin](https://grafana.com/docs/grafana/latest/developers/plugins/package-a-plugin)

Let me know if you run into any issues! I _think_ the linter error will go away by following the steps above. Otherwise, I'll make sure it gets fixed. Thanks!

**Comment by marcusolsson** [comment]
Awesome! I put in the new URL into the [plugin validator](https://grafana-plugins-web-vgmmyppaka-lz.a.run.app/?url=https://github.com/simPod/GrafanaJsonDatasource/releases/download/v0.2.3/simpod-json-datasource.zip) and it found a few issues.

Screen Shot 2021-01-27 at 10 49 25

This should be the last of it :)

**Comment by simPod** [comment]
🚢

**Comment by marcusolsson** [comment]
Published! 🎉
